.history/src/google_takeout_metadata/processor_20250906093332.py

```
@@ -0,0 +1,246 @@
+        # Update the title field
+        json_data['title'] = new_image_path.name
+        
+        # Write updated JSON to new location
+        with open(new_json_path, 'w', encoding='utf-8') as f:
+            json.dump(json_data, f, indent=2, ensure_ascii=False)
+        
+        # Remove old JSON file
+        json_path.unlink()
+        logger.info("Updated and renamed JSON: %s to %s", json_path, new_json_path)
+        
+        return new_image_path, new_json_path
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)